src/audio/pipeline.c

```diff
@@ -91,6 +91,10 @@ struct pipeline *pipeline_new(struct sof_ipc_pipe_new *pipe_desc,
 	schedule_task_init(&p->pipe_task, type, pipe_desc->priority,
 			   pipeline_task, p, pipe_desc->core, 0);
 
+#if CONFIG_PERF_MEASURE
```

Comment:
Rather than having more conditional code in pipeline, the measure_ APIs should be declared as static inline skeletons when measure is disabled by Kconfig.

Reply:
Right